Casks/apogee-duet.rb

```diff
@@ -9,19 +9,49 @@
 
   depends_on macos: '>= :mavericks'
 
-  app 'Duet USB Firmware Updater.app'
   pkg 'Duet Software Installer.pkg'
 
-  uninstall pkgutil:   'com.apogee.pkg.*',
+  uninstall pkgutil:   [
+                         'com.apogee.pkg.ApogeePopup',
+                         'com.apogee.pkg.ApogeeServices',
+                         'com.apogee.pkg.DuetUSB*',
+                         'com.apogee.pkg.Maestro2',
+                       ],
             launchctl: [
                          'com.ApogeePopup.plist',
                          'com.DuetUSBDaemon.plist',
                        ],
+            quit:      [
+                         'com.apogee.Apogee-Maestro-2',
```

Comment:
```suggestion
                         'com.apogee.Apogee-Maestro-.*',
```

As we don’t want to forget to update the version number in the future.